docs-src/community.scrbl

```diff
@@ -0,0 +1,35 @@
+#lang scribble/manual
+@(require "lib.rkt")
+@(mint-scope "Community")
+
+@title[#:version reach-vers #:tag "community"]{Community}
+
+Below are links to Reach-related instructional content from the Reach community. To submit a link for consideration, contact mhagen@"@"reach.sh.
+
+@itemlist[
+
+@item{
+  @link["https://github.com/hagenhaus/market-day"]{Market Day} example app by Matt Hagen on July 20, 2021.
+}
+
+@item{
+  @link["https://devtimeblog.medium.com/connecting-funding-an-algorand-test-wallet-from-a-svelte-reach-application-6e3166462436"]{Connecting/Funding an Algorand (test) Wallet from a Svelte + Reach Application} by Krysztof on July 5, 2021.
+}
+
+@item{
+  @link["https://devtimeblog.medium.com/connecting-funding-an-algorand-test-wallet-from-a-angular-reach-application-93999a8c23d3"]{Connecting/Funding an Algorand (test) Wallet from a Angular + Reach Application} by Krysztof on July 5, 2021.
+}
+
+@item{
+  @link["https://devtimeblog.medium.com/connecting-funding-an-algorand-test-wallet-from-a-vanillajs-parcel-reach-application-54f9fc996d53"]{Connecting/Funding an Algorand (test) Wallet from a VanillaJS + Parcel + Reach Application} by Krysztof on June 28, 2021.
+}
+
+@item{
+  @link["https://devtimeblog.medium.com/connecting-funding-an-algorand-test-wallet-from-a-reach-react-application-e6affaadcf65"]{Connecting/Funding an Algorand (test) Wallet from a Reach + React Application} by Krysztof on May 29, 2021.
+}
+
+@item{
+  @link["https://devtimeblog.medium.com/the-reach-algorand-bounty-hack-c46e8b823003"]{The Reach Algorand Bounty Hack} by Krysztof on May 25, 2021.
```

Comment:
Please reformat so the dates are in a sane format (2021/05/25) and at the front.